src/main/resources/assets/paths/tdrV2DraftDAR.yaml

```diff
@@ -0,0 +1,27 @@
+post:
+  summary: createDraftDataAccessRequest
+  description: Create Draft Data Access Request
+  parameters:
+    - name: identifiers
+      in: query
+      description: |
+        Dataset identifiers to be included in the Data Access Request (comma separated). e.g. DUOS-123456, DUOS-123457
+        Datasets are restricted to those for which the user has a Library Card with a Data Access Agreement for the dataset's Data Access Committee
+      required: true
+      schema:
+        type: string
+    - name: projectTitle
+      in: query
+      description: Project title for the new Data Access Request.
+      required: false
+      schema:
+        type: string
+  tags:
+    - TDR
+  responses:
```

Comment:
Do we need to add the 400 response for the error thrown if the user does not have all of the DAAs on their LC for any of the datasets in this draft DAR creation?

Reply:
We do!